crates/symbolicator/src/services/symbolication/comparisons.rs

```diff
@@ -15,6 +15,7 @@ pub(super) enum NewStackwalkingProblem {
 
 fn fixup_modules(modules: &mut [(DebugId, RawObjectInfo)]) {
     modules.sort_by_key(|module| module.0);
+    modules.sort_by_key(|module| module.1.image_addr);
```

Comment:
I think you can just sort by `(module.0, module.1.image_addr)` ? Tuples should implement `Ord` in an appropriate way I guess?